### DISCUSSION TITLE
Spell and Basic Attack Stat Scaling Improvement

### DISCUSSION THREAD

**Discussion by Uyarrr** [AscensionGameDev/Intersect-Engine#1510] [Ideas]
1. Allow to scale more than one stat.  
"I think its something we really need. Example: Stomp spell, takes 50% scaling from DEF and 100% scaling from ATK. I think its pretty useful and required. Same for items (basic attacks).
2. Scale from Integer Player Variables and support for more than one.
"In my opinion, this will be one of the best features ever, allowing items and spells scale from player variables will allow developers to make more complicated stuff much easier without modifying source code!

**Comment by AisenGaming** [comment]
I fully support this idea. This would be extremely useful

**Comment by WeylonSantana** [comment]
This idea is awesome!

**Comment by Uyarrr** [comment]
Bump!

**Comment by jcsnider** [comment]
Please don't bump. It doesn't make things happen quicker, or move the PR higher in the list, all it does is spam those following the project with emails.
